pages/wiki/emulator.md

```diff
@@ -42,3 +42,41 @@ The previous command sets up a port forwarding so that you can connect to the em
 ```
 ssh -p 2222 ceres@localhost
 ```
+
+# Troubleshooting
+Here are some common problems and their solutions.
+
+## `qemu-system-i386: warning: host doesn't support requested feature: CPUID.80000001H:EDX.lm [bit 29]`
+This just means that you are running an i386 emulation on a 64-bit host processor.  For AsteroidOS, this will make no difference, but you can run `qemu-system-x86-64` instead of `qemu-system-i386` to bypass the error, which has to do with [long mode](https://en.wikipedia.org/wiki/Long_mode) on x86-64 machines.
```

Comment:
Nowadays everyone uses x86-64 machines anyway. I think it'd be better to make `qemu-system-x86-64` the default in the existing instructions